config/settings.yml

```diff
@@ -6,10 +6,7 @@ mail:
   et:
     england_wales: ETHelpwithfees@hmcts.gsi.gov.uk
     scotland: GLASGOWET@hmcts.gsi.gov.uk
-  dwp_notification_alert:
-    - dan.thompson@digital.justice.gov.uk
-    - kathryn.kram@justice.gov.uk
-    - sustainingteamsupport@hmcts.net
+  dwp_notification_alert: <%= ENV['DWP_NOTIFICATION_ALERT_EMAILS'] %>
```

Comment:
What happens if a developer does not have this set ?
Also,  assuming we are using a .env file, should we not add it to there as well ?

Reply:
@garytaylor this is called by rake task so nothing happens. We are setting this value in deploy repo so it's with other ENV values.